src/components/JsonSchemaValidator/Schemas/dbtAttribution_0.1.0.js

```diff
@@ -0,0 +1,228 @@
+      longDescription: 'If `false`, only considers the channel at the start of the session (i.e. first page view). If `true`, considers multiple channels in the conversion session as well as historically.',
+      packageDefault: 'false',
+      title: 'Consider Intrasession Channels',
+    },
+    snowplow__spend_source: {
+      title: 'Conversions Source',
+      description: 'Source of marketing spend table',
+      longDescription: 'The source (schema and table) of your marketing spend source. Optional, needed for the ROAS calculation of the snowplow_attribution_overview. Should be changed to a table reference with `spend` by `channel` and/or `campaign` by `spend_timestamp` (which denotes a timestamp field) information.',
+      type: 'string',
+      packageDefault: "{{ source('atomic', 'events') }}",
+      group: 'Warehouse and Tracker',
+    },
+    snowplow__conversion_stitching: {
+      type: 'boolean',
+      group: 'Contexts, Filters, and Logs',
+      longDescription: 'This should align with the variable of the same name inside the unfied package. If allowed it will consider the stitched_user_id field, not the user_identifier in the source data for more accurate results.',
+      packageDefault: 'false',
+      title: 'Conversion Stitching',
+    },
+    snowplow__conversion_clause: {
+      type: 'string',
+      title: 'Conversions Clause',
+      group: 'Operation and Logic',
+      longDescription: 'A string of sql to filter on certain conversion events.',
+      packageDefault: 'cv_value > 0',
+    },
+    snowplow__attribution_list: {
+      type: 'array',
+      description: '> Click the plus sign to add a new entry',
+      minItems: 1,
+      title: 'Attribution List',
+      longDescription: 'List of attribution types to use for reporting. Can be at least one of: first_touch, last_touch, linear, position_based).',
+      packageDefault: '[`first_touch`, `last_touch`, `linear`, `position_based`]',
+      group: 'Contexts, Filters, and Logs',
+      items: { type: 'string' },
+    },
+    snowplow_attribution_start_date: {
+      type: 'string',
+      format: 'date',
+      title: 'Attribution Start Date',
+      group: 'Operation and Logic',
+      longDescription: 'The date to start processing events from in the package on first run or a full refresh, based on the cv_tstamp (conversion timestamp).',
+      packageDefault: '2023-01-01',
+      description:
+        'The date to start processing events from in the package on first run or a full refresh, based on `cv_tstamp`',
+    },
+    snowplow__enable_paths_to_non_conversion: {
+      type: 'boolean',
+      group: 'Contexts, Filters, and Logs',
+      longDescription: 'If `true`, enable the paths_to_non_conversion model, which is a drop and recompute table that may be needed for more in-depth attribution analysis (used in the `path_summary` table as well',
+      packageDefault: 'false',
+      title: 'Enable Paths To Non Conversion',
+    },
```

Comment:
Ah this is where the closing bracket should be!

Reply:
whatever this was i think this was resolved by the ' changes